src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -9,7 +9,8 @@ public static class MetricEventNames
         public const string HostStartupLatency = "host.startup.latency";
 
         // function level events
-        public const string FunctionInvokeByTriggerFormat = "function.invoke.{0}";
+        public const string FunctionBindingTriggerTypeFormat = "function.binding.{0}";
```

Comment:
nit: since this isn't just for triggers, maybe rename to `FunctionBindingTypeFormat`?